tests/security/cc/cc_selinux_setup.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Setup "audit-test" test environment of a system running SELinux
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93441
+
+use base 'selinuxtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    select_console "root-console";
+
+    my $file = "selinux-policy-targeted-20210419-5.2.noarch.rpm";
+    assert_script_run "wget --quiet " . data_url("selinux/$file");
+    assert_script_run("rpm -ivh --nosignature --nodeps --noplugins $file");
+
+    # Set SELINUXTYPE=targeted
+    # TODO: "targeted" type still has some problems (reboot failed)
```

Comment:
Again, please don't add TODOs in source code, use an issue tracker for that.

Reply:
Done.